src/Google/Ads/GoogleAds/V0/Util/GoogleAdsErrors.php

```diff
@@ -0,0 +1,118 @@
+<?php
```

Comment:
You still put the file in the wrong directory.
It must be GoogleAds/Util/V0.

Reply:
Sorry, I was mistaken, I just realized that this folder was the one for generated code. Fixed it.